pkg/importer/writer.go

```diff
@@ -165,6 +165,39 @@ func buildQueryString(params []Param) string {
 	return query
 }
 
+func buildResponseContentString(responses []Response) string {
+	if len(responses) == 0 {
+		return ""
+	}
+
+	contentsString := " [responses=["
+	responses = removeResponseDuplicates(responses)
+	for _, val := range responses {
+		if val.Content.contentType != "" && val.Content.name != "" {
+			contentsString = contentsString + "[" +
+				val.Content.contentType[10:len(val.Content.contentType)-1] + "\", \"" + val.Content.name + "\"], "
+		}
+	}
+	contentsString = contentsString[:len(contentsString)-2]
+	contentsString += "]]"
+	if contentsString == " [responses]]" {
+		return ""
+	}
+	return contentsString
+}
+
+func removeResponseDuplicates(responses []Response) []Response {
+	keys := make(map[Content]bool)
+	list := []Response{}
+	for _, entry := range responses {
+		if _, value := keys[entry.Content]; !value {
```

Comment:
`if _, value := keys[entry.Content]; !value {` 
This line is slightly ambiguous; usually the second value of accessing a map this way is a bool of if the key `entry.Content` is in the map, here you're calling this the `value`, even though the value would be the first value:
`if value, ok:= keys[entry.Content]; !ok` ...
If you're checking for if the duplicate is in the map then you should be using an empty struct instead of a bool

Reply:
1- I care about the second variable and thus I named it value. Will change it as it creates confusion. 
2- I'm checking if !value and appending the values to a list and that is why bool is sufficient and I need to use an empty struct.

Reply:
1- I was using the bool only and that is why I chose to name it value. I see that it creates confusion so I will rename it. 

2- I am using the bool to build a map which consists of the Content as keys and the bool as value to see if it exists or not and then I append the response to the list. So I don't see why I need to use a struct?